db/migrate/20170207173837_set_service_display_and_retired_to_false.rb

```diff
@@ -0,0 +1,12 @@
+class SetServiceDisplayAndRetiredToFalse < ActiveRecord::Migration[5.0]
+  Service = Class.new(ActiveRecord::Base)
```

Comment:
Service model has `type` column so we need to add `inheritance_column = = :_type_disabled` # disable STI

```
  class Service < ActiveRecord::Base
    self.inheritance_column = :_type_disabled # disable STI
  end
```